scylla/src/transport/session_test.rs

```diff
@@ -2857,3 +2858,33 @@ async fn test_manual_primary_key_computation() {
         .await;
     }
 }
+
+#[tokio::test]
+async fn test_non_existent_dc_return_correct_error() {
+    let ks = "iot";
+
+    let host = "127.0.0.1";
```

Comment:
The tests fail because they run Scylla on different IP. Please take a look at other tests in this file - they use `create_new_session_builder()` to get a session builder with correct IP. You can then call 
```
        .default_execution_profile_handle(handle)
        .build()
        .await
        .expect("cannot create session");
```
on resulting builder.

Reply:
Thanks